### PR TITLE
fix(docs): update design of home page

### PR DESCRIPTION
**Pull request by varnikajain15**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: docs/src/pages/HomeCard.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+};
+
+const HomeCard = () => {
+  const menuSection = useHomeMenu();
+
+  return (
+    <section className="px-12 py-11 home-menu">
+      <Row>
+        {menuSection.map((menuItem, key) => {
+          console.log(menuItem);
```
Comment (anuradha9712) [inline docs/src/pages/HomeCard.js]: Remove console from here

File: docs/src/pages/HomeCard.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          const [isHovered, setIsHovered] = useState(false);
+          return (
+            <Column size={3} key={key}>
+              <div
+                className="mr-7 h-100 overflow-visble"
```
Comment (anuradha9712) [inline docs/src/pages/HomeCard.js]: Use `overflow-visible` instead of `overflow-visble`

File: docs/src/pages/home/HomeCard.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+const MenuIcons = ({ name }) => {
+  const SvgIcons = HomeIcons[name] || (() => <div></div>);
+  return <SvgIcons />;
+};
+
+const HomeCard = () => {
+  const menuSection = useHomeMenu();
+  const [isHovered, setIsHovered] = useState('');
+  return (
+    <section className="home-card-padding-y home-menu">
```
Comment (veekays) [inline docs/src/pages/home/HomeCard.js]: Please follow BEM to name CSS classes

File: docs/src/pages/home/HomeCard.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      <Row>
+        <Column size={1} />
+        {menuSection.map((menuItem, key) => {
+          return (
+            <Column size={2.5} key={key} className={`${key === 3 ? 'mr-0': 'mr-7'}`}>
```
Comment (veekays) [inline docs/src/pages/home/HomeCard.js]: `Column` component do not support size="2.5"

File: docs/src/pages/home/HomeCard.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+import * as HomeIcons from '../../util/HomeIcons';
+
+const MenuIcons = ({ name }) => {
+  const SvgIcons = HomeIcons[name] || (() => <div></div>);
+  return <SvgIcons />;
+};
+
+const HomeCard = () => {
+  const menuSection = useHomeMenu();
+  const [isHovered, setIsHovered] = useState('');
```
Comment (veekays) [inline docs/src/pages/home/HomeCard.js]: Please rename `isHovered` & `setIsHovered` according to its functionality

File: docs/src/pages/home/HomeCard.js (hunk 3 above)
Comment (varnikajain15) [reply docs/src/pages/home/HomeCard.js]: done

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**